standard/local_currency/local_currency.lua

```diff
@@ -0,0 +1,47 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:LocalCurrency
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local LocalCurrencyData = mw.loadData('Module:LocalCurrency/Data')
+
+local LocalCurrency = {}
+
+local USD = 'usd'
+local USD_TEMPLATE_ALIAS = '1'
+
+function LocalCurrency.template(frame)
+	local args = Arguments.getArgs(frame)
+	local currencyCode = args.currency or args[1]
+	if currencyCode == USD_TEMPLATE_ALIAS then
+		currencyCode = USD
+	end
+	local prizeValue = args.prizepool or args[2]
+	return LocalCurrency.display(currencyCode, prizeValue, {setVariables = true})
+	-- should we add a default fallback here (like the current template)?
```

Comment:
Sure, if the plan is only calling this function from the template, having its behavior to be the same as the template does make sense.

Reply:
added and retested stuff